configs/components/cpp-pcp-client.rb

```diff
@@ -15,10 +15,6 @@
   pkg.build_requires "leatherman"
 
   if platform.is_aix?
-    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-gcc-5.2.0-1.aix#{platform.os_version}.ppc.rpm"
```

Comment:
Removed gcc, cmake, boost from all components that depend on Leatherman since it includes them. This makes it easier to rev versions of those packages safely.